lms/djangoapps/experiments/utils.py

```diff
@@ -78,6 +78,23 @@
     flag_undefined_default=True
 )
 
+# .. toggle_name: experiments.deprecated_metadata
+# .. toggle_type: feature_flag
+# .. toggle_default: True
+# .. toggle_description: Toggle for using the deprecated method for calculating user metadata
+# .. toggle_category: experiments
+# .. toggle_use_cases: monitored_rollout
+# .. toggle_creation_date: 2019-5-8
+# .. toggle_expiration_date: None
+# .. toggle_warnings: None
+# .. toggle_tickets: REVEM-350
+# .. toggle_status: supported
+DEPRECATED_METADATA = WaffleFlag(
```

Comment:
out of curiosity is this flag here because we're not able to confirm locally that the new code functions as intended?

Reply:
I did test it locally (and it seems fine to me). But historically bugs in user-metadata creation code have only been found by people who are not me exercising it on prod. So I'm paranoid :-)